### DISCUSSION TITLE
Can we use regular expressions in tag-mapping.xml and and render theme.xml

### DISCUSSION THREAD

**Discussion by ravivarmahattaraki** [mapsforge/mapsforge#1228] [Q&A] [answered]
Can we use regular expressions in tag-mapping.xml and and render theme.xml
for example
<rule e="any" k="RCID" v="^([1-9]|1[0-9])$">
        <symbol src="jar:symbols/bench.svg" />
    </rule>
![image](https://user-images.githubusercontent.com/30779078/124876393-94b10280-dfe7-11eb-9abf-82858dec514c.png)

**Comment by devemux86** [comment, marked as answer]
See the [render theme documentation](https://github.com/mapsforge/mapsforge/blob/master/docs/Rendertheme.md), the [xml schemas](https://github.com/mapsforge/mapsforge/tree/master/resources) and the default themes / tag-mapping for the available rules.
